### PR TITLE
feat: generate notebook from template

### PR DESCRIPTION
**Pull request by drdelambre**
This allows us to generate any number of random notebooks from a template. You just add the template to the giant object, and it registers a route you can reference with a link (no user info in there like orgs or what not). should be able to deeplink from docs as well, but try it as a signed out user before shipping

Peek 2021-05-20 11-32

turns out there's a bug with the youtube component where autoplay doesnt fire when we deeplink to the page, so please ignore

### CODE REVIEW

File: src/flows/context/flow.list.tsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ export function hydrate(data) {
 export const FlowListProvider: FC = ({children}) => {
   const [flows, setFlows] = useLocalStorageState('flows', DEFAULT_CONTEXT.flows)
   const [currentID, setCurrentID] = useState(DEFAULT_CONTEXT.currentID)
-  const {orgID} = useParams<{orgID: string}>()
```
Comment (drdelambre) [inline src/flows/context/flow.list.tsx:140]: boooooo

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 51 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1474)

File: src/flows/components/FromTemplatePage.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    })
+  }, [add, history, org.id, loading, setLoading, template])
+
+  return <div />
+}
+
+const FromTemplatePage: FC = () => {
+  return (
+    <FlowListProvider>
+      <div />
```
Comment (Asalem1) [inline src/flows/components/FromTemplatePage.tsx:71]: What divs? (See what i did there?)

**Review by Asalem1 — APPROVED** [review]
Not sure why I don’t like the setLoading(false) useEffect

File: src/flows/components/FromTemplatePage.tsx (hunk 2 above)
Comment (drdelambre) [reply src/flows/components/FromTemplatePage.tsx:71]: WHOOPS

**Review by mavarius — APPROVED** [review]
Thanks for this, you absolute wizard!

**Pull request merged**